CSS/dropdown.css

```diff
@@ -0,0 +1,13 @@
+.dropdown {
```

Comment:
A bit more comments in the top of each style file explaining the purpose of the file would be really nice.   This makes it easier for a developer to fully understand exactly what that file is intended to be used for.   This way when they are adding features they can correctly determine if they should create a new style file or use the existing.